.github/workflows/deployment_tests.yml

```diff
@@ -49,51 +50,67 @@ jobs:
           cat folders.yaml
 
       - name: Deploy Google folder
+        id: deploy-folders
         run: |
           cd google_folder
           terraform init
           terraform plan -out="./plan.tfplan"
           terraform apply plan.tfplan
-          terraform plan -destroy -out="./destroy.tfplan"
-          terraform apply destroy.tfplan
+
+      - name: Destroy Google Folders
+        if: steps.deploy-folders.outcome != 'skipped'
+        run: |
+          cd google_folder
+          terraform destroy -auto-approve
+      
 
       # Deploy google projects
       - name: Add IDs to GCP Project Files
         env:
           GOOGLE_CREDENTIALS: ${{secrets.GCP_SA_KEY}}
         run: |
           cd google_project/resource
-          sed -i "s/<project_id>/${{secrets.GCP_PROJECT_ID}}-$(date +%y%m%d%H%M)//" projects.yaml
+          sed -i "s/<project_date>/$DATE/g" projects.yaml
```

Comment:
Changed this as project names were specified in the projects.yaml. 
If more secrecy is required can update can go back to using the secrets, append a number to different projects for uniqueness